contracts/EmergencyActivationMultisig.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.23;
+
+import {RestrictedMultisigBase} from "./RestrictedMultisigBase.sol";
+
+interface IEmergencyProtectedTimelock {
+    function emergencyActivate() external;
+}
+
+contract EmergencyActivationMultisig is RestrictedMultisigBase {
+    uint256 public constant EMERGENCY_ACTIVATE = 1;
+
+    address emergencyProtectedTimelock;
+
+    constructor(
+        address _owner,
+        address[] memory _members,
+        uint256 _quorum,
+        address _emergencyProtectedTimelock
+    ) RestrictedMultisigBase(_owner, _members, _quorum) {
+        emergencyProtectedTimelock = _emergencyProtectedTimelock;
+    }
+
+    function voteEmergencyActivate() public onlyMember {
```

Comment:
i'd either not call this `vote` or rename the `vote` in the `EmergencyExecutionMultisig` since they have a diff semantics (this one approved, the other one allows to approve or disapprove)

e.g. name this `approveEmergencyActivate` and leave the other one `voteExecuteProposal`